compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Base.hs

```diff
@@ -796,6 +796,13 @@ data Template = Template
   }
   deriving (Eq, Data, Generic, NFData, Show)
 
+-- | Definition of an exception type.
+data DefException = DefException
+  { dexLocation :: !(Maybe SourceLoc)
+  , dexName :: !TypeConName
```

Comment:
I'm wondering if `exn` (like in OCaml) or `exc` would be a better prefix. But I don't care deeply.

Reply:
I was using `dexc` but it was really messy. I think `dexn` is good though, I'll use that.

Reply:
Actually just `exn` :-)